plugin/plugin.go

```diff
@@ -81,11 +81,11 @@ func NewCLIPlugin(ctx context.Context, name, path string) (*CLIPlugin, error) {
 	if err != nil {
 		// Ignore any file which we cannot Stat
 		// (e.g. due to permissions or anything else).
-		return nil, err
+		return nil, fmt.Errorf("plugin instantiation failed because the executable file is either not found or inaccessible: %w", err)
```

Comment:
hmmm, this doesn't seem to be correct?
Function `NewCLIPlugin` can be called in various places not necessarily by `plugin install`. Therefore, its error message should not include details like `plugin instantiation failed`.

Reply:
`Instantiation` refers to creating an instance of the CLI plugin object, which is what NewCLIPlugin does. I think it's not related to plugin install, is it?

Reply:
My bad. I thought it was `installation`. Let's change it to:
`fmt.Errorf("plugin executable file is either not found or inaccessible: %w", err)`

Reply:
Updated.